social/event/v0.1.0/schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/social/event/v0.1.0/schema.json",
+    "title": "Event",
+    "description": "Schema for a calendar event",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4285F4",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "social_event",
+        "indexes": {
+            "name": ["name"],
+            "startTime": ["start.dateTime", "start.date"],
+            "endTime": ["end.dateTime", "end.date"],
+            "source": ["sourceApplication", "sourceId"]
+        }
+    },
+    "layouts": {
+        "create": ["name", "start", "end", "location"],
+        "view": ["name", "description", "start", "end", "location", "organizer", "attendees", "status"]
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/v0.1.0/schema.json"
+        },
+        {
+            "properties": {
+                "status": {
+                    "title": "Status",
+                    "description": "Status of the event",
+                    "type": "string",
+                    "enum": ["confirmed", "tentative", "cancelled"]
+                },
+                "description": {
+                    "title": "Description",
+                    "description": "Detailed description of the event",
+                    "type": "string"
+                },
+                "location": {
+                    "title": "Location",
+                    "description": "Location of the event",
+                    "type": "string"
+                },
+                "creator": {
+                    "title": "Creator",
+                    "description": "Details of the event creator",
+                    "$ref": "https://common.schemas.example.com/profile/person/v0.1.0/schema.json"
+                },
+                "organizer": {
+                    "title": "Organizer",
+                    "description": "Details of the event organizer",
+                    "$ref": "https://common.schemas.example.com/profile/person/v0.1.0/schema.json"
+                },
+                "start": {
+                    "title": "Start",
+                    "description": "Start time of the event",
+                    "type": "object",
+                    "properties": {
+                        "dateTime": {
+                            "type": "string",
+                            "format": "date-time"
+                        },
+                        "date": {
+                            "type": "string",
+                            "format": "date"
+                        },
+                        "timeZone": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "end": {
```

Comment:
Make `start.date` and `end.date` required.
 
 Populate them from `dateTime` if not specified in the event. We can then just use `date` property for indexes to save disk space and make more efficient.

Reply:
> Make `start.date` and `end.date` required.
> 
> Populate them from `dateTime` if not specified in the event. We can then just use `date` property for indexes to save disk space and make more efficient.

Fixed